<!--
  1 failure:  :disappointed_rel...
  3 warning:  :mag: Can't find ..., cli/lib/tasks/pro..., cli/lib/tasks/pro...
  1 messages
  
  DangerID: danger-id-Danger;
-->

<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  :disappointed_relieved: `npm test` failed. See below for details.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  :mag: Can't find junit reports at `./junit.*.xml`, skipping generating JUnit Report.
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **cli/lib/tasks/process-js-task.js#L157** - cli/lib/tasks/process-js-task.js line 157 – Avoid calling back inside of a promise. (promise/no-callback-in-promise)
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **cli/lib/tasks/process-js-task.js#L164** - cli/lib/tasks/process-js-task.js line 164 – Avoid calling back inside of a promise. (promise/no-callback-in-promise)
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  ```
> titanium-mobile@8.1.0 test /Users/build/jenkins/workspace/ium-sdk_titanium_mobile_PR-11024
> npm run ios-sanity-check && grunt


> titanium-mobile@8.1.0 ios-sanity-check /Users/build/jenkins/workspace/ium-sdk_titanium_mobile_PR-11024
> ./build/scons check-ios-toplevel

Running "appcJs:src:lintOnly" (appcJs) task

Running "eslint:src" (eslint) task

/Users/build/jenkins/workspace/ium-sdk_titanium_mobile_PR-11024/cli/lib/tasks/process-js-task.js
  119:69  error    Missing semicolon                       semi
  157:12  warning  Avoid calling back inside of a promise  promise/no-callback-in-promise
  164:5   warning  Avoid calling back inside of a promise  promise/no-callback-in-promise

/Users/build/jenkins/workspace/ium-sdk_titanium_mobile_PR-11024/iphone/cli/commands/_build.js
  5884:15  warning  Avoid calling back inside of a promise                                  promise/no-callback-in-promise
  6411:79  warning  'out' is defined but never used. Allowed unused args must match /^_.+/  no-unused-vars

/Users/build/jenkins/workspace/ium-sdk_titanium_mobile_PR-11024/iphone/cli/hooks/frameworks.js
   60:5   warning  Expected catch() or return                  promise/catch-or-return
   60:34  warning  Avoid calling back inside of a promise      promise/no-callback-in-promise
  425:5   warning  Each then() should return a value or throw  promise/always-return

✖ 8 problems (1 error, 7 warnings)
  1 error, 0 warnings potentially fixable with the `--fix` option.

Warning: Task "eslint:src" failed. Use --force to continue.

Aborted due to warnings.
npm ERR! Test failed.  See above for more details.

```
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against cb9b1742f1ec1516dec9c8b410f5a20a29cec8b4
</p>

--- a/cli/lib/tasks/process-js-task.js
+++ b/cli/lib/tasks/process-js-task.js
@@ -116,7 +116,7 @@ class ProcessJsTask extends IncrementalFileTask {
 		this.jsFiles = this.data.jsFiles;
 		this.jsBootstrapFiles.splice(0, 0, ...this.data.jsBootstrapFiles);
 		return Promise.all(Object.keys(this.jsFiles).map(relPath => {
-			return limit(() => this.processJsFile(this.jsFiles[relPath].src))
+			return limit(() => this.processJsFile(this.jsFiles[relPath].src));
 		}));
 	}
 

